README.md

```diff
@@ -97,7 +97,33 @@ Many browsers have started blocking cross-origin or "third party" cookies by def
   * [token.getWithoutPrompt](#tokengetwithoutpromptoptions) must have access to cookies on the Okta domain via an iFrame running on your application's page.
   * [token.renew](#tokenrenewtokentorenew) uses [token.getWithoutPrompt](#tokengetwithoutpromptoptions) and is subject to the same limitations.
 
-If your application depends on any of these methods, you should try to either rewrite your application to avoid using these methods or communicate to your users that they must enable third party cookies. Okta engineers are currently working on a better long-term solution to this problem.
+Notably, a browser blocking 3rd party cookies prevents automatic token renewal from working.   The code will behave as if the user's session at the issuer site has expired.
+
+#### How to handle Third party cookie issues
+
+Note that third party cookie blocking is controlled by the browser.  `okta-auth-js` calls run in the browser can be affected, but `okta-auth-js` calls made on the server cannot.  These solutions assume you are running `okta-auth-js` from the browser.
+
+##### Solution 1: Use a custom domain URL  
+  * If the domain of your app and the domain that holds the session cookie are the same domain, there is no "third" party to trigger issues with third-party cookies
+  * You can learn more about how to use this feature here: https://developer.okta.com/docs/guides/custom-url-domain/overview/
+  * After making those changes, you will need to make sure your app is configured to use the matching url
+  * This should prevent third-party cookie issues
+
+##### Solution 2: Rely on redirects
+  * If the user is redirected to the issuer site for any transaction that requires a session cookie, and redirected back to the application once that is complete, there is never a "third" party interaction to trigger issues with third-party cookies
+  * When token renewal fails, direct the user to login via OIDC redirect (as they did originally), either by sending them to an okta-hosted sign-in page or your application's sign-in page that starts an OIDC redirect authentication
+  * You can detect token renewal failure by catching the rejected promise returned by (for example) [token.getWithoutPrompt()](https://github.com/okta/okta-auth-js#example)
```

Comment:
Catching the promise rejection will only work if the app is manually renewing tokens. For most applications, with `autoRenew` set to true they will not have access to the promise. They can listen to the "error" event from TokenManager. However, with the new authState logic this will not be necessary or recommended. There will be a new authState event which will be fired when the renew fails. isAuthenticated will be false.